android/src/com/mapswithme/maps/widget/placepage/PlacePageView.java

```diff
@@ -849,6 +856,9 @@ private void refreshDetails(@NonNull MapObject mapObject)
     String website = mapObject.getMetadata(Metadata.MetadataType.FMD_WEBSITE);
     String url = mapObject.getMetadata(Metadata.MetadataType.FMD_URL);
     refreshMetadataOrHide(TextUtils.isEmpty(website) ? url : website, mWebsite, mTvWebsite);
+    String wikimedia_commons = mapObject.getMetadata(Metadata.MetadataType.FMD_WIKIMEDIA_COMMONS);
+    String wikimedia_commons_text =  TextUtils.isEmpty(wikimedia_commons) ? "" : "WIKIMEDIA COMMONS";
```

Comment:
Does mapObject.getMetadata already checks for null and replaces it with an empty string? Also relevant to similar calls above.

Reply:
TextUtils.isEmpty checks for null.